search-app/docs/api.md

```diff
@@ -1798,9 +1798,9 @@ Find collections matching tool concept id.
 ##### <a name="c-polygon"></a> Polygon
 
 Polygon points are provided in counter-clockwise order. The last point should match the first point to close the polygon. The values are listed comma separated in longitude latitude order, i.e. lon1, lat1, lon2, lat2, lon3, lat3, and so on.
-This parameter supports the and/or option as shown below.
+The polygon parameter could be either "polygon" or "polygon[]", it supports the and/or option as shown below. The first example is to search for collections that match both the first polygon and the second polygon. The second example is to search for collections that match either the first polygon or the second polygon.
```

Comment:
This is unclear. When there is multiple polygons, you can NOT use `polygon` parameter to search.